src/components/views/settings/tabs/user/PreferencesUserSettingsTab.tsx

```diff
@@ -356,19 +263,6 @@ export default class PreferencesUserSettingsTab extends React.Component<IProps,
                     { this.renderGroup(PreferencesUserSettingsTab.SPACES_SETTINGS, SettingLevel.ACCOUNT) }
                 </div>
 
-                <div className="mx_SettingsTab_section">
-                    <span className="mx_SettingsTab_subheading">{ _t("Communities") }</span>
-                    <p>{ _t("Communities have been archived to make way for Spaces but you can convert your " +
-                        "communities into Spaces below. Converting will ensure your conversations get the latest " +
-                        "features.") }</p>
-                    <details>
-                        <summary>{ _t("Show my Communities") }</summary>
-                        <p>{ _t("If a community isn't shown you may not have permission to convert it.") }</p>
-                        <CommunityMigrator onFinished={this.props.closeSettingsFn} />
```

Comment:
@nadonomy are we happy to get rid of the Community -> Space migrator in this very near future?

Reply:
for public context: Synapse plans to remove the endpoints soonish, so the migrator will cease with that. This functionality additionally needs to go in order to unblock other work.

Reply:
We can point to riots.im (effectively community maintained), or we might be able to spin up an `old.element.io` instance or something?

Reply:
Actually, we'll need to keep the APIs around if we plan to publish an old version. We can probably delay removal of the API endpoints on the synapse side which makes the migrator possible, but I don't imagine Synapse will want to keep them around for much longer. 

Keeping the migrator code in the app prevents further work, unfortunately.

Reply:
@nadonomy is this resolved to your satisfaction?